pkgs/applications/virtualization/spim/default.nix

```diff
@@ -0,0 +1,77 @@
+{ mkDerivation, lib, fetchsvn, qmake, qtbase, qttools, bison, flex }:
+
+mkDerivation rec {
+  pname = "QtSpim";
+  version = "722";
+
+  src = fetchsvn {
+    url = "https://svn.code.sf.net/p/spimsimulator/code";
+    rev = version;
+    sha256 = "1hfz41ra93pdd2pri5hibl63sg9yyk12a8nhdkmgj7h9bwgqxw6b";
+  };
+
+  sourceRoot = "code-r${version}/QtSpim";
+
+  nativeBuildInputs = [ bison flex qmake ];
+
+  buildInputs = [ qtbase qttools ];
+
+  # Remove build artifacts from the repo
+  preConfigure = ''
+    rm parser_yacc.h
+    rm parser_yacc.cpp
+    rm scanner_lex.cpp
+
+    rm help/qtspim.qhc
+  '';
+
+  # Disable documentation building, because qtspim can't find compiled docs
+  patches = [ ./qtspim-disable-docs.patch ];
+
+  # Commented out to save some build time (uncomment after fixing doc finding)
+  # # Fix bug in a generated Makefile
+  # postConfigure = ''
+  #   substituteInPlace Makefile --replace '$(MOVE) help/qtspim.qhc help/qtspim.qhc;' ""
+  # '';
+
+  # buildPhase = ''
+  #   export QT_PLUGIN_PATH="${qtbase.bin}/${qtbase.qtPluginPrefix}"
+  #   export QT_QPA_PLATFORM_PLUGIN_PATH="${qtbase.bin}/${qtbase.qtPluginPrefix}/platforms"
+  #   make
+  # '';
+
+  installPhase = ''
+    install -sDm0755 QtSpim $out/bin/qtspim
+
+    install -D ../Documentation/qtspim.man $out/usr/share/man/man1/qtspim.1
+    gzip -f --best $out/usr/share/man/man1/qtspim.1
+
+    install -d $out/usr/share/qtspim
```

Comment:
All of these paths should be `$out/share/...` (no `/usr` prefix). This may affect finding the files at runtime.

Reply:
Thanks, fixed!

Finally found what's up with the docs -- doesn't have anything to do with Qt, just a bunch of hardcoded paths in the source code (missed initially them while stracing as qtspim `access`ed them, while I expected other syscalls).